save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -353,6 +371,8 @@ class DockerService(
     companion object {
         private val log = LoggerFactory.getLogger(DockerService::class.java)
         private val loggingContext = LoggingContextImpl(log)
+        private const val AGENTS_START_SLEEP_INTERVAL = 10_000L
+        private const val AGENTS_START_TIMEOUT = 60_000L
```

Comment:
Let's move it to `ConfigProperties`. This way it will be easier to tune this logic without recompiling and restarting an application